doc/source/user/configoptions.rst

```diff
@@ -411,6 +411,31 @@ General options
 
       Controls whether curved geometries should be approximated by linear geometries.
 
+- .. config:: OGR_ORGANIZE_POLYGONS
+     :choices: DEFAULT, SKIP, ONLY_CCW, CCW_INNER_JUST_AFTER_CW_OUTER
+
+     Defines the method used to classify polygons as holes or shells. Although
+     one of the options is named ``DEFAULT``, some drivers may default to a
+     different method to reduce processing by taking advantage of a format's
+     constraints. The following methods are available, in order of decreasing
+     expected runtime:
+
+     - ``DEFAULT``: perform a full analysis of the topological relationships
+       between all polygons, classifying them as shells or holes and associating
+       them according to the OGC Simple Features convention.
+
+     - ``ONLY_CCW``: only attempt to classify polygons with a counter-clockwise
+       orientation, assuming that all polygons with a clockwise orientation
+       represent holes. This is the convention of the Shapefile format, whose
+       driver defaults to this method.
+
+     - ``CCW_INNER_JUST_AFTER_CW_OUTER``: Assume inner rings (holes) have
+       counter-clockwise orientation and immediately follow the
+       clockwise-oriented outer ring with which they are associated.
+
+     - ``SKIP``: avoid attempting to classify polygons as shells or holes. A
+       single polygon will be returned with all polygons as top-level polygons.
```

Comment:
```suggestion
       single multipolygon (or multisurface or geometrycollection, depending on if there are curve polygons or a mix of regular and curve polygons) will be returned with all polygons as top-level polygons.
```

Reply:
A mix of regular and curved polygons would produce a MultiSurface, no? My read was that the GeometryCollection is returned if a line/point finds its way in, or if the polygons do not form a valid multipolygon (crossing edges)

Reply:
indeed, you're correct.

Reply:
Updated with "A single geometry (MultiPolygon or MultiCurve) will be returned with all polygons as top-level polygons."

Reply:
MultiCurve cannot be emitted. The 4 possiblye types are:
- Polygon
- MultiPolyon
- MultiSurface
- GeometryCollection: if presence of non-polygon in input or invalid topology